gridstatus/miso.py

```diff
@@ -679,8 +679,586 @@ def _get_generation_outages_data(
             .reset_index(drop=True)
         )
 
+    @support_date_range(frequency="DAY_START")
+    def get_binding_constraints_supplemental(
+        self,
+        date: str | pd.Timestamp,
+        end: str | pd.Timestamp = None,
+        verbose: bool = False,
+    ) -> pd.DataFrame:
+        """Get the supplemental binding constraints data from MISO.
+
+        Source URL: https://www.misoenergy.org/markets-and-operations/real-time--market-data/market-reports/#nt=%2FMarketReportType%3ADay-Ahead%2FMarketReportName%3ABinding Constraints Supplemental (xls)&t=10&p=0&s=MarketReportPublished&sd=desc
+
+        Args:
+            date (str | pd.Timestamp): Start date
+            end (str | pd.Timestamp, optional): End date. Defaults to None.
+            verbose (bool, optional): Verbosity. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: Supplemental binding constraints data
+        """
+        query_date = date - pd.Timedelta("1D")
```

Comment:
Is this a day ahead dataset? Should we add `day_ahead` to the method name?

Reply:
I'm going by the Data Spec https://www.notion.so/gridstatus/Binding-Constraints-Supplemental-130e835f42aa801da5abf4e91dbf10e9?pvs=4